pkgs/development/tools/language-servers/postgres-lsp/default.nix

```
@@ -6,17 +6,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "postgres-lsp";
-  version = "unstable-2024-01-11";
+  version = "unstable-2024-03-24";
```

```suggestion
  version = "0-unstable-2024-03-24";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming